packages/11ty/_includes/components/figure/table/index.js

```diff
@@ -9,13 +9,11 @@ const path = require('path')
 module.exports = function(eleventyConfig) {
   const figureCaption = eleventyConfig.getFilter('figureCaption')
   const figureLabel = eleventyConfig.getFilter('figureLabel')
+  const figureTableElement = eleventyConfig.getFilter('figureTableElement')
```

Comment:
As we are already in the `figure` module here it may be clear enough to rename this `tableElement`